server/controllers/events/events_controller.go

```diff
@@ -300,9 +301,14 @@ func (e *VCSEventsController) HandleGithubCommentEvent(event *github.IssueCommen
 		}
 	}
 
+	eventTimestamp := time.Now()
+	githubComment := event.Comment
+	if githubComment != nil && githubComment.CreatedAt != nil {
+		eventTimestamp = *githubComment.CreatedAt
+	}
```

Comment:
can event.Comment or event.Comment.CreatedAt really be nil? I'd be ok with not checking this and addressing if we see failures, because then we'll at least know what we're preventing.

Reply:
Wasn't clear to me when scanning through Github docs, but there is a unit test that checks that scenario:
https://github.com/lyft/atlantis/blob/release-v0.17.3-lyft.1/server/controllers/events/events_controller_test.go#L159

Reply:
discussed offline and determined we can keep the nil checks in :)